query-engine/query-engine-wasm/build.sh

```diff
@@ -19,32 +20,41 @@ fi
 
 echo "Using build profile: \"${WASM_BUILD_PROFILE}\"" 
 
-# Check if wasm-pack is installed
 if ! command -v wasm-pack &> /dev/null
 then
     echo "wasm-pack could not be found, installing now..."
     curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh
 fi
 
+echo "ℹ️  Configuring rust toolchain to use nightly and rust-src component"
+rustup default nightly-2024-01-25 
+rustup target add wasm32-unknown-unknown
+rustup component add rust-src rust-std --target wasm32-unknown-unknown
+
+# export RUSTFLAGS="-Zlocation-detail=none"
```

Comment:
Is this comment a leftover? Should we actually include it?

Reply:
No, we shouldn't yet. Doesn't contribute to lower sizes. We are still working on this sorry. Will move it back to draft, and only ask for feedback when it's ready.